### ISSUE TITLE
Piece of text treated as separate lines when selected

### ISSUE THREAD

**Issue by jeremypress** [mozilla/pdf.js#9984]
I'm interested to see why PDF.js is interpreting this piece of text as multiple (lined) blocks? It's either a browser text layer issue, or maybe based on PDF.js' adherence to the spec? Here's the extracted data from Brendan's browser:

```
 /P <</MCID 0>> BDC BT
/F1 12 Tf
1 0 0 1 72.025 708.6 Tm
0 g
0 G
[(w)-3(w)-3(w)-3(.as)-2(ld)5(fkd)4(s)-4(lf)4(k)-2(.c)8(o)-3(m)-2(/lon)12(gh)6(o)-3(r)5(n)] TJ
ET
BT
1 0 0 1 225.05 708.6 Tm
[(-)] TJ
ET
BT
1 0 0 1 228.67 708.6 Tm
[(s)-4(o)-3(lo)-2(r)] TJ
ET
BT
1 0 0 1 253.05 708.6 Tm
[(-)] TJ
ET
BT
1 0 0 1 256.67 708.6 Tm
[(au)4(s)-4(ton?)6(s)-4(tar)7(t)-8(3)] TJ
ET
BT
1 0 0 1 324.45 708.6 Tm
[( )] TJ
ET
 EMC  /P <</MCID 1>> BDC BT
1 0 0 1 72.025 693.97 Tm
[( )] TJ
ET
```

Attach (recommended) or Link to PDF file here:
[download.pdf](https://github.com/mozilla/pdf.js/files/2298143/download.pdf)
^ this PDF was generated by Microsoft Office

Configuration:
- Web browser and its version:
latest chrome
- Operating system and its version:
MacOS Sierra
- PDF.js version:
demo
- Is a browser extension:
no

Steps to reproduce the problem:
1. Copy the whole line
2. Paste into a text editor, notice that the line is split up into many

What is the expected behavior? (add screenshot)
I would expect a piece of text like this to be captured as one line 
What went wrong? (add screenshot)
<img width="414" alt="screen shot 2018-08-17 at 10 26 46 am" src="https://user-images.githubusercontent.com/2474027/44279931-17f26600-a208-11e8-8ed5-bd75aaa8f1a8.png">

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by timvandermeij** [comment]
This improved slightly after the fix above. It now copies as:

```
www.asldfkdslfk.com/longhorn
-solor
-auston?start3
```

**Comment by timvandermeij** [comment]
Fixed by #10197.
